src/pages/about.tsx

```diff
@@ -77,11 +85,90 @@ const AboutPage: React.FC = () => {
         </GridItem>
         <GridItem xs={12} md={8}>
           <AboutSubTitle>Team</AboutSubTitle>
-          <div>TODO: TeamCards</div>
         </GridItem>
       </Grid>
+      <ImageCardGrid>
+        <GridItem xs={6} sm={3}>
```

Comment:
On the samllest `sm`, the images are quite small. I would therefore go for `sm=4` and `md=3`.

Reply:
I don't agree. It looks exactly as in the design.
And as long as we not know exactly how the pictures will look like, I would roll with it as it is.